docs-e2e-testing/e2e/jest-preview.cy.js

```diff
@@ -1,7 +1,41 @@
 describe('Tests for Jest Preview', () => {
-  it('Check Jest Preview website is alive', () => {
+  beforeEach( function() {
     cy.visit('https://www.jest-preview.com/')
     cy.contains('h1', 'Jest Preview')
-    cy.contains('Debug your Jest tests. Effortlessly.')
+  })
+
+  it('Check CTA Get Started button if it works', () => {
+    cy.get('.button').click()
+    cy.url()
+      .should('contain', 'docs/getting-started/intro')
+    cy.contains('h1','Introduction')
+  })
+
+  it('Check Docs page is present', () => {
+    cy.contains("Docs").click()
+    cy.url()
+      .should('contain', 'docs/getting-started/intro')
+    cy.contains('h1','Introduction')
+  })
+  
+  it('Check Blog page is present', () => {
+    cy.contains("Blog").click()
+    cy.url()
+      .should('contain', '/blog')
+    cy.contains('Recent posts')
+  })
+  
+  it('Check API page is present', () => {
+    cy.contains("API").click()
+    cy.url()
+      .should('contain', '/api')
+    cy.contains('Getting Started')
+  })
+  
+  it('Check Demo link is correct and Demo page is present', () => {
+    cy.contains('Demo').invoke('removeAttr', 'target').click()
+    cy.url()
+      .should('include', 'stackblitz.com/edit/jest-preview')
+    cy.contains('Jest Preview')
```

Comment:
@huyenuet With this style of writing tests. Cypress needs to visit https://www.jest-preview.com **5** times.
Instead, you can visit it only **1** time, then navigate and assert 5 times. That would make your tests run much faster (especially in a CI environment)